### PR TITLE
add cuav x7plus

### PR DESCRIPTION
**Pull request by cuhome**
add cuav x7plus

### CODE REVIEW

File: common/source/docs/common-cuav-x7plus-overview.rst [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+GPIOs
+=====
+The 14 outputs can be used as GPIOs (relays, buttons, RPM etc). To use them you need to set the output’s SERVOx_FUNCTION to -1. See GPIOs page for more information.
+
+The numbering of the GPIOs for PIN variables in ArduPilot is:
+
+pins:
+
+PWM1(M1) 50
```
Comment (Hwurzburg) [inline common/source/docs/common-cuav-x7plus-overview.rst:117]: same comment as above regarding bullets and no blank lines

File: common/source/docs/common-cuav-x7plus-overview.rst [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+PWM7（M7) 56
+
+PWM8(M8) 57
+
+Analog inputs
+=============
+The CUAV X7+/X7+ Pro has 2 analog inputs, one 6V tolerant and one 3.3V tolerant
+
+ADC Pin18 -> ADC 6.6V Sense
```
Comment (Hwurzburg) [inline common/source/docs/common-cuav-x7plus-overview.rst:137]: and here again

File: common/source/docs/common-cuav-x7plus-overview.rst [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+Analog 3.3V RSSI input pin = 6
+
+
+Battery Monitor
+===============
+
+The autopilot includes a DroneCAN power module and battery monitor, :ref:`common-can-pmu`. Connect to one of the CAN ports and its already pre-configured for use.
+If you are using an analog battery monitor, connect to the Power A connector and set the following parameters:
+
+BATT_MONITOR=4；
```
Comment (Hwurzburg) [inline common/source/docs/common-cuav-x7plus-overview.rst:150]: and here

File: common/source/docs/common-cuav-x7plus-overview.rst [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+
+.. youtube:: jiAjyL_ruec
+    :width: 100%
+    
+More Information
+================
+
+- `CUAV docs <http://doc.cuav.net/flight-controller/x7/en/x7.html>`__
+- `x7 schematic <https://github.com/cuav/hardware/tree/master/X7_Autopilot>`__
+- [copywiki destination="plane,copter,rover,blimp"]
```
Comment (Hwurzburg) [inline common/source/docs/common-cuav-x7plus-overview.rst:182]: wrong formatting
```suggestion

[copywiki destination="plane,copter,rover,blimp"]
```

File: common/source/docs/common-cuav-x7plus-overview.rst [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+- SERIAL0 = console = USB
+- SERIAL1 = Telemetry1 = USART1
+- SERIAL2 = Telemetry2 = USART6
+- SERIAL3 = GPS1 = USART1
+- SERIAL4 = GPS2 = UART4
+- SERIAL5 = USER = UART8 
+- SERIAL6 = USER = UART7 (DEBUG TX/RX)
+
+Serial protocols can be adjusted to personal preferences.
+
```
Comment (Hwurzburg) [inline common/source/docs/common-cuav-x7plus-overview.rst:92]: needs an RC input section stating where to connect, which protocols are supported on that pin (PPM,SBUS,DSM,etc.), and that serial RX like CRSF,FPort, SRLX2, etc need to use a full UART

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
while CI does not catch it, this causes build errors...removing the old doc from TOC causes the error...
you should just replace the old X7 page with this one as an update to it....

**Comment by cuhome** [comment]
@Hwurzburg I have done the modification, please help me to check it again.

File: common/source/docs/common-autopilots.rst [hunk 6]
```diff
@@ -17,8 +17,8 @@ Open hardware
     CUAV V5 Plus <common-cuav-v5plus-overview>
     CUAV V5 Nano <common-cuav-v5nano-overview>
     CUAV Nora <common-cuav-nora-overview>
-    CUAV X7/ X7 Pro <common-cuav-x7-overview>
     CUAV Pixhawk v6X <common-cuav-pixhawkv6X>
+    CUAV X7/X7Pro/X7+/X7+ Pro <common-cuav-x7-family-overview>
```
Comment (cuhome) [inline common/source/docs/common-autopilots.rst:21]: I don't find the common-cuav-x7-family-overview file in the folder, is it missing?

Comment (Hwurzburg) [reply common/source/docs/common-autopilots.rst:21]: sorry pushed it to my repo instead of yours...there now

**Comment by cuhome** [comment]
Except that the usb description needs to be modified, I think it can be merged.

**Pull request merged**